definitions/tools/pvacfuse.wdl

```diff
@@ -27,7 +27,7 @@ task pvacfuse {
 
   Int space_needed_gb = 10 + round(size([input_fusions_zip], "GB") * 3)
   runtime {
-    docker: "griffithlab/pvactools:3.0.0"
+    docker: "laljorani20/pvactools:latest"
```

Comment:
We should switch this back to an official release before merging.

Reply:
I believe this is blocked on @susannasiebert switching back to pvactools effort and creating a release. But yeah we don't want to be pointing to personal docker images like this.  I have been meaning to search the whole repo and see if we are doing it anywhere else.

Reply:
I just released pVACtools 3.1.0 that has the combinatorial class II alleles enabled.